### PR TITLE
you can get more arrays, as a treat

### PR DESCRIPTION
**Pull request by Glitchy404**

### CODE REVIEW

**Comment by Glitchy404** [comment]
wait i forgot the commas

File: arrays.py [hunk 1]
```diff
@@ -134,11 +134,15 @@
...
     "PC players",
+    "Nintendo players",
     "Dani",
     "Mara",
     "Rail",
     "Taavi",
     "Herobrine",
+    "Linux Ricers",
+    "Sackboy"
+    "Robloxians"
```
Comment (OniriCorpe) [inline arrays.py:145]:
```suggestion
    "Sackboy",
    "Robloxians",
```

File: arrays.py [hunk 2]
```diff
@@ -421,4 +425,16 @@
...
+    "a singular robuck",
+    "a singular ticket",
+    "score bubbles",
+    "prize bubbles",
+    "vbucks",
+    "collectabells",
+    "free alternative",
+    "linux rice",
+    "more ram",
+    "more arrays"
```
Comment (OniriCorpe) [inline arrays.py:439]:
```suggestion
    "more arrays",
```

**Review by OniriCorpe — COMMENTED** [review]
still missing some commas ^w^

**Comment by theresnotime** [comment]
Hope you don't mind that I fixed it up :3 (after breaking it a little more!) — thank you for the additions!

**Pull request merged**